turbo.json

```diff
@@ -26,15 +26,24 @@
       "dependsOn": ["^build", "codegen:entrypoints", "codegen:mergeRouters"],
       "outputs": [".next/**", "dist/**", "build/**"]
     },
+    "@examples/minimal-react#build": {
+      "cache": false
+    },
```

Comment:
temp fix - nested build folders weren't cached properly for this example. will investigare later